redirects.json

```diff
@@ -126,6 +126,26 @@
       "name": "CSS reverse-proxy",
       "match": "^css/(.*)",
       "destination": "https://primer.github.io/css/{R:1}"
+    },
+    {
```

Comment:
@langermank these redirects will need to move to the `redirects` array further up this file. They are currently in the rewrites section, so won't get processed correctly.